### PR TITLE
[E3DB-1139] Move E3DB Java Hosting to Maven Central

### PR DESCRIPTION
**Pull request by efabens**
- Update gradle publish configurations to publish to Maven Central
- Remove gradle publish configurations to publish to self-hosted maven
- Update README to reflect new process to publish artifacts

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -193,13 +201,14 @@ ext {
...
-Assuming your AWS account has the correct privileges, publish by running the following task:
+Assuming your OSSRH account has the correct privileges, publish by running the following task:
 
 ```bash
-$ ./gradlew :publish:publishRemote
+$ ./gradlew :publish:publishMavenCentral
 ```
 
-The task will publish both the plain Java JAR and the Android AAR to the maven repository.
+The task will publish both the plain Java JAR and the Android AAR to the maven stagin repository, actual deployment needs to be done with the [Nexus Repository Manager](https://oss.sonatype.org/#stagingRepositories.)
```
Comment (galxy25) [inline DEVELOPMENT.md]: `staging`

File: publish/android/build.gradle [hunk 2]
```diff
@@ -100,3 +109,57 @@ dependencies {
...
+
+artifacts {
+  archives javadocJar, sourceJar
+}
+
+uploadArchives {
+  group = "com.tozny.e3db"
+  archivesBaseName = "e3db-client-android"
+  version = project(":publish").ext.version
+  println()
```
Comment (galxy25) [inline publish/android/build.gradle]: still needed?

File: publish/build.gradle [hunk 3]
```diff
@@ -30,8 +30,8 @@ task publishLocal(dependsOn: [":publish:android:publishMavenAarPublicationToMave
   }
 }
 
-task publishRemote(dependsOn: [":publish:android:publishMavenAarPublicationToMavenRepository", ":publish:plain:publishMavenPublicationToMavenRepository"]) {
+task publishMavenCentral(dependsOn: [":publish:android:uploadArchives", ":publish:plain:uploadArchives"]) {
   doLast {
-    println "Published remote artifacts."
+    print ln "Published Artifacts to Nexus Repository Manager"
```
Comment (galxy25) [inline publish/build.gradle]: Should this be `println`?

**Review by galxy25 — APPROVED** [review]

**Review by lkwdwrd — APPROVED** [review]
looks good, this will make e3db so much more accessible!

**Pull request merged**